internal/pkg/db/config.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 EMQ Technologies Co., Ltd.
```

Comment:
You can use your name or company. This applies to all your changes

Reply:
Oki I will change it then

Reply:
@ngjaying Ok I adjusted codebase, please check correctness of it. In case of such stuff I am far from confidence.

Reply:
It looks great.